src/wallet/rpc/transactions.cpp

```diff
@@ -447,7 +447,7 @@ RPCHelpMan listtransactions()
                         {RPCResult::Type::OBJ, "", "", Cat(Cat<std::vector<RPCResult>>(
                         {
                             {RPCResult::Type::BOOL, "involvesWatchonly", /*optional=*/true, "Only returns true if imported addresses were involved in transaction."},
-                            {RPCResult::Type::STR, "address", "The bitcoin address of the transaction."},
+                            {RPCResult::Type::STR, "address",  /*optional=*/true, "The bitcoin address of the transaction (not returned if the transaction contains OP_RETURN null data)."},
```

Comment:
I don't think this is true. A scriptpubkey can be nonstandard, but that doesn't imply it has an OP_RETURN

Reply:
Also (unrelated): A *transaction* does not have an address. At most it is the address of the output

Reply:
Is the following message better ?
`not returned if the output address cannot be decoded (such as OP_RETURN null data).`

Reply:
I think "does not have an address" is more correct. "cannot be decoded" implies that there is a correct way to encode addresses in a transaction, but that's not correct as addresses are not encoded into transactions, they are turned into scripts.

Reply:
Done in https://github.com/bitcoin/bitcoin/pull/26349/commits/eb679a7896ce00e322972a011b023661766923b9.